provider/cmd/pulumi-resource-docker-build/schema.json

```diff
@@ -412,11 +412,11 @@
         },
         "url": {
           "type": "string",
-          "description": "The cache server URL to use for artifacts.\n\nDefaults to `$ACTIONS_RUNTIME_URL`, although a separate action like\n`crazy-max/ghaction-github-runtime` is recommended to expose this\nenvironment variable to your jobs.",
```

Comment:
Curious why is this changing.

Reply:
Fixed in https://github.com/pulumi/pulumi-docker-build/pull/89#discussion_r1621018357